edb/server/protocol/ai_ext.py

```diff
@@ -1125,15 +1126,27 @@ async def _start_chat(
     model_name: str,
     messages: list[dict[str, Any]],
     stream: bool,
+    temperature: Optional[float],
```

Comment:
Let's make these keyword arguments.

Reply:
hey @elprans can u pls tell me do u mean all arg should be positional or only everything after http_client for example:
  ```
  protocol: protocol.HttpProtocol,
    request: protocol.HttpRequest,
    response: protocol.HttpResponse,
    provider: ProviderConfig,
    http_client: http.HttpClient,
    *,
    model_name: str,
    messages: list[dict[str, Any]],
    ....
```

Reply:
I would make everything keyword